src/shared/components/VideoCategoryCard/FeaturedVideoCategoryCard.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+
+import { useHover } from '@/hooks/useHover'
+import { SvgVideoCategoriesScienceAndTechnology } from '@/shared/icons/VideoCategoriesScienceAndTechnology'
+import { sizes, transitions } from '@/shared/theme'
+
+import {
+  FeaturedContainer,
+  FeaturedContent,
+  FeaturedIconCircle,
+  FeaturedVideoText,
+  FeaturedVideoTitleContainer,
+  PlayerContainer,
+} from './FeaturedVideoCategoryCard.style'
+
+import { SkeletonLoader } from '../SkeletonLoader'
+import { Text } from '../Text'
+import { VideoPlayer } from '../VideoPlayer'
+
+export type FeaturedVideoCategoryCardProps = {
+  variant?: 'default' | 'compact'
+  loading?: boolean
+  color: string
+} & React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>
```

Comment:
Why do we need those details props here?

Reply:
It's nice to have i.e. supports className and style props for the container div

Reply:
I think it's more clean to just include `className` directly as we do for other components, I don't think we should need any other prop, for example we shouldn't use the `style` prop directly